roles/gitlab_runner/defaults/main.yml

```
@@ -49,3 +49,5 @@ gitlab_runner_session_server_timeout: 1800
 gitlab_runner_autoscaler_plugin_version: "0.21.1"
 gitlab_runner_autoscaler_plugin_url: "https://github.com/sardinasystems/fleeting-plugin-openstack/releases/download/{{ gitlab_runner_autoscaler_plugin_version }}/fleeting-plugin-openstack_{{ gitlab_runner_autoscaler_plugin_version }}_linux_amd64.tar.gz"
 gitlab_runner_autoscaler_plugin_checksumfile: "https://github.com/sardinasystems/fleeting-plugin-openstack/releases/download/{{ gitlab_runner_autoscaler_plugin_version }}/fleeting-plugin-openstack_{{ gitlab_runner_autoscaler_plugin_version }}_sha512-checksums.txt"
+
+gitlab_runner_butane_config: "butane-config.bu.j2"
```

Could it be also `butane-config.bu` without the template fileending and use that one throughout the role?

```suggestion
gitlab_runner_butane_config: "butane-config.bu"
```